src/handbook/customer/sales/subscription-agreement-1.5.njk

```diff
@@ -487,6 +487,10 @@ The SaaS Software will be available to Customer 99.5% of each calendar month com
 <p>
 Subject to the exclusions herein, “Downtime” is defined as any period of time during which Customer is unable to access the SaaS Software. “Downtime” specifically excludes: (i) scheduled maintenance, which is conducted between 9:00 PM (PT) and 4:00  AM (PT); (ii) Customer’s systems, networks, equipment, or connections; (iii) Customer’s acts other than in accordance with the Agreement, including, without limitation, any negligence, willful misconduct or use of the SaaS Software in breach of this Agreement; or (iv) Force Majeure - circumstances beyond FlowFuse’s reasonable control including, without limitation, acts of any governmental body, war, insurrection, sabotage, embargo, fire, flood, strike or other labor disturbance, unavailability of or interruption or delay in telecommunications or third party services, failure of third party software or inability to obtain supplies used in or equipment needed for provision of the SaaS Software.
 </p>
+<p>
+Customer agrees to the following limits on MQTT broker usage:
+The maximum size of any retained message is 10 MB, a rate limit of 500 messages per second, and a bandwidth limit of 10mb per second per client.
```

Comment:
And can we agree on higher limits? Or is that not per tenant? Are these defaults?
```suggestion
The maximum combined sizes of all retained message is 10 MB, a rate limit of 500 messages per second, and a bandwidth limit of 10mb per second per client.
```

Reply:
@hardillb Can you confirm that ZJ's suggestion here is accurate? That the combined size of all retained messages is 10MB, rather than that the size of any individual retained message is 10MB?

Reply:
No, the retained message size limit is per message, the edit is incorrect,

The limits are per client, not per tenant. It would be acceptable to have 10 clients all publishing at 10mbs per second.

The described limits are lower than the system will handle, but they are set so we can come down from to them if needed. Not all limits are currently enforced, these were picked to give us room to change things if needed without having to change the ToCs